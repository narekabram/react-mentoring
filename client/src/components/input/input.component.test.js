import React from 'react';
import { shallow } from 'enzyme';
import Input from './index';

describe('Input', () => {
    it('Adds Input snapshot testing', () => {
        const component = shallow(<Input/>);

        expect(component).toMatchSnapshot();
    });
});
