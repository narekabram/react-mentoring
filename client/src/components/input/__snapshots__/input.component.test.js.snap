// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Adds Input snapshot testing 1`] = `ShallowWrapper {}`;
